src/provisioner/log.py

```diff
@@ -0,0 +1,44 @@
+#!/bin/env python3
+
+# CORTX Python common library.
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+
+import os
+from cortx.utils.log import Log
+
+
+class CortxProvisionerLog(Log):
+    """ Cortx provisioner log """
+
+    logger = None
+
+    @staticmethod
+    def init(service_name, log_path=None, console_output=False):
+        """ Initialize log """
+
+        if not log_path:
+            log_path = os.path.join('/var/log/cortx', 'provisioner')
+
+        log_level = os.getenv('CORTX_PROVISIONER_DEBUG_LEVEL', 'INFO')
```

Comment:
Log level should come as a parameter to this class. Caller can decide if it can come from ENV or command line.

Reply:
Done.

Reply:
As part of this, now cortx-setup command is having optional argument -l to set custom log level.
@ujjwalpl